app/CocoaPods/CPHomeWindowController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  CPHomeWindowController.h
+//  CocoaPods
+//
+//  Created by William Kent on 9/10/15.
+//  Copyright © 2015 CocoaPods. All rights reserved.
+//
```

Comment:
Please remove these file copyright notices, as it just gets messy in the long run with regards to attribution in an OSS context.